training/src/main/java/com/wolox/training/models/User.java

```diff
@@ -40,8 +44,11 @@
   @Column(nullable = false)
   private LocalDate birthdate;
 
-  @OneToMany(mappedBy = "user")
-  private List<Book> books;
+  @ManyToMany(cascade = CascadeType.ALL)
+  @JoinTable(name = "user_book",
+  joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+  inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"))
+  private List<Book> books = new ArrayList<>();
```

Comment:
Some comments about this association:
- That new ArrayList is better to do it inside a constructor (To encapsulate this initialization behaviour). Maybe redefine the default constructor public Book() and instance that ArrayList
- The name of the table, the joinColumns, the inverseJoinColumns and its referencedColumnName are not needed. I think they will work fine with the default values :) You can try it
- This cascade type is the used on the baeldung example but is not the right one for this case. This is because, for example, it will cause that the remove of an user provokes the remove of a book. The persist type will cause a similar problem